### PR TITLE
Andy/group tile

### PR DESCRIPTION
**Pull request by andylouisqin**
Need to collaborate with the backend team to figure out the image overlay trick, but otherwise all components are in place for the group listings page

### CODE REVIEW

File: app/views/groups/index.html.erb [hunk 1]
```diff
@@ -1,53 +1,52 @@
...
-<%= link_to "Create new group", new_group_path %>
+      <% if group.users.count == 1 %>
+        <h6><%= group.users.count %> member</h6>
+      <% else %>
+        <h6><%= group.users.count %> members</h6>
+      <% end %>
+
+      <div class="member-display">
+        <% group.users.last(3).each do |user| %>
+          <span data-tooltip aria-haspopup="true" class="has-tip" title=<%= user.first_name + "&nbsp;" + user.last_name %>><%= image_tag(user.photo(:thumb)) %></span>
```
Comment (hidesminimally) [inline app/views/groups/index.html.erb]: @andylouisqin Merge with master if you haven't and then do 
`
 <span data-tooltip aria-haspopup="true" class="has-tip" title=<%= user.decorate.full_name %>><%= image_tag(user.photo(:thumb)) %></span>
`

File: app/views/groups/index.html.erb [hunk 2]
```diff
@@ -1,53 +1,52 @@
...
+            <!-- TODO (Shimmy): Add user.full_name decorator -->
```
Comment (hidesminimally) [inline app/views/groups/index.html.erb]: @andylouisqin delete this comment afterwards

File: db/seeds.rb [hunk 3]
```diff
@@ -6,35 +6,39 @@
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
 
+def seed_image(file_name)
+  File.open(File.join(Rails.root, "/app/assets/images/seed/#{file_name}"))
```
Comment (hidesminimally) [inline db/seeds.rb:10]: nice

File: app/assets/stylesheets/ashby_settings.scss [hunk 4]
```diff
@@ -1,20 +1,45 @@
...
+$ashby-dark-green: #417500;
+$ashby-red: #DE6868;
+$ashby-dark-red: #90091A;
 $ashby-tan: #F7F5F2;
 
 // FONT SIZES
 $h1-heading-size: 64px;
 $h2-heading-size: 48px;
-$h3-heading-size: 32px;
+$h3-heading-size: 24px;
```
Comment (hidesminimally) [inline app/assets/stylesheets/ashby_settings.scss:17]: @andylouisqin weren't you going with multiples by 16?

File: app/assets/stylesheets/_group_tile.scss [hunk 5]
```diff
@@ -0,0 +1,48 @@
+@import "ashby_settings.scss";
```
Comment (hidesminimally) [inline app/assets/stylesheets/_group_tile.scss:1]: @andylouisqin spacing is inconsistent! should be two spaces throughout all code

File: app/assets/stylesheets/_navbar.scss [hunk 6]
```diff
@@ -11,10 +11,14 @@
 }
 
 .top-bar .my-profile img {
```
Comment (hidesminimally) [inline app/assets/stylesheets/_navbar.scss:13]: @andylouisqin spacing is inconsistent! should be two spaces throughout all code

File: app/assets/stylesheets/ashby_settings.scss [hunk 7]
```diff
@@ -1,20 +1,45 @@
...
 $body-font-size: 16px;
 $subtext-font-size: 14px;
 
+h1 {
```
Comment (hidesminimally) [inline app/assets/stylesheets/ashby_settings.scss:21]: @andylouisqin spacing is inconsistent! should be two spaces throughout all code

File: app/views/groups/index.html.erb (hunk 1 above)
Comment (andylouisqin) [inline app/views/groups/index.html.erb]: @lilshim Just added this, but I'm getting only the first name for each user. Thoughts?
screen shot 2015-11-16 at 8 25 36 pm

File: app/assets/stylesheets/ashby_settings.scss (hunk 4 above)
Comment (andylouisqin) [inline app/assets/stylesheets/ashby_settings.scss:17]: Actually the least common multiple is 8 😉

**Pull request merged**